tools/jinja2-filters.py

```diff
@@ -18,3 +21,19 @@ def filter_b64decode(encoded_value, return_type = "string"):
         return b64decode(encoded_value, validate = True)
     else:
         raise UnsupportedDataType("The encoded value is an unsupported data type")
+
+def filter_chained_get(dictionary_name, chain):
+    if not isinstance(dictionary_name, dict):
+        raise NotADict("The variable does not a dict")
```

Comment:
This does not do what I meant. Here's how it should work in edge cases:

Let's assume that `config_values` is a dict that looks like this:
``` yaml
a:
  b:
    d: 2
    e: "string" 
    f:
    g: {}
  c:
```

The expected results are:
- `config_values | changed_get("a.b.d")` = `2`
- `config_values | changed_get("a.b.d.x")` - error
- `config_values | changed_get("a.b.f")` - `null`
- `config_values | changed_get("a.b.f.x")` - `null`
- `config_values | changed_get("a.b.f.g")` - `{}`
- `config_values | changed_get("a.b.f.g.x")` - `null`
- `config_values | changed_get("a.b")` - `{'d': 2, 'e': "string", 'f': None, 'g': {}}`